releases/release-1.31/release-notes/maps/pr-124352-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 124352
+releasenote:
+  text: Fixed a race condition in kube-controller-manager and scheduler caused by
```

Comment:
```suggestion
  text: Fixed a race condition in kube-controller-manager and the scheduler, caused by
```

Reply:
Based on style guide, I am not sure if we need back ticks for kube-controller-manager
@npolshakova  please confirm ?

Reply:
We should use proper markdown formatting in the release notes. So use backticks for CLIs, flags, etc. This one's tricky because although kube-controller-manager is a component tool, the [official documentation page](https://kubernetes.io/docs/concepts/overview/components/) doesn't use backticks so I think it's fine leaving it off.

Reply:
ok, thanks both :)